### PR TITLE
fix producing nuget packages during the build

### PR DESCRIPTION
**Pull request by krwq**

### CODE REVIEW

File: scripts/build/build.ps1 [hunk 1]
```diff
@@ -12,7 +12,7 @@ $ErrorActionPreference="Stop"
 
 . "$PSScriptRoot\..\common\_common.ps1"
 
-_ "$RepoRoot\scripts\build\generate-version.ps1"
+. "$RepoRoot\scripts\build\generate-version.ps1"
```
Comment (piotrpMSFT) [inline scripts/build/build.ps1]: Can you break up this change? There is WAY too much going on here for a single PR!

**Comment by piotrpMSFT** [comment]
:shipit:

File: scripts/build/build.ps1 (hunk 1 above)
Comment (brthor) [inline scripts/build/build.ps1]: :smile: But seriously what is this magic?

Comment (krwq) [inline scripts/build/build.ps1]: underscore was too long, I changed it to the dot, @piotrpMSFT are two dots ok?

Comment (krwq) [inline scripts/build/build.ps1]: @brthor the "_" is equivalent to "call" (in this repo) where "." is equivalent to include. generate-version does set bunch of variables which get lost when you call the script and return, including it preserves them

Comment (brthor) [inline scripts/build/build.ps1]: Ah makes sense now. Took that comment out because I started looking it up. 

The `_` function is pretty neat

Comment (krwq) [inline scripts/build/build.ps1]: it's super nice :smiley: Improves my powershell experience by a factor of 2 - that was the most annoying thing for me in powershell and this is nicely fixing it :+1:

**Pull request merged**